material-overrides/home.html

```diff
@@ -2,81 +2,115 @@
+            <p class="framer-text">Start building</p>
+          </div>
         </a>
       </div>
     </div>
-    <div>
-      <img
-        src="./assets/images/hero.webp"
-        alt="kluster.ai Hero"
-        class="hero-image"
-      />
+  </div>
+
+  <div class="feature-cards">
+    <div class="feature-card">
+      <div class="feature-header">
+        {% include ".icons/octicons/zap-16.svg" %}
+        <h2>Real-time <br /> Inference</h2>
+      </div>  
+      <p>Instant predictions backed by autoscaling infrastructure. Built for latency-sensitive LLM applications.</p>
+      <a href="get-started/start-building/real-time/" class="card-button">Learn More</a>
+    </div>
+
+    <div class="feature-card">
+      <div class="feature-header"> 
+        {% include ".icons/octicons/list-unordered-16.svg" %}
+        <h2>Batch<br /> Inference</h2>
+      </div> 
+        <p>Process large datasets with parallel inference at scale. Ideal for precomputing outputs or automating bulk tasks.</p>
+      <a href="/get-started/start-building/batch/" class="card-button">Run Batch Job</a>
+    </div>
+
+    <div class="feature-card">
+      <div class="feature-header"> 
+        {% include ".icons/octicons/tools-16.svg" %}
+        <h2>Fine-Tune<br /> Models</h2>
+      </div>
+      <p>Refine models with your data to reduce costs and boost efficiency, without sacrificing accuracy.</p>
+      <a href="/tutorials/klusterai-api/finetuning-sent-analysis/" class="card-button">Fine-Tune a Model</a>
+    </div>
+        
+    <div class="feature-card">
+      <div class="feature-header"> 
+        {% include ".icons/octicons/mortar-board-16.svg" %}
+        <h2>Verify</h2>
+      </div> 
+      <p>Not all LLM responses are equal. Use Verify to assess quality and reliability before taking action or showing results to users.</p>
+      <a href="/get-started/verify/overview/" class="card-button">Explore Verify</a>
     </div>
   </div>
-  <div class="featured-content">
-    <div class="feature">
-      <h2>Get an API key</h2>
-      <img src="./assets/images/circle1.webp" alt="Colab notebook" />
-      <p>
-        Obtain your API key to access the
-        <a href="https://kluster.ai" target="\_blank">kluster.ai</a>
-        platform and start building and manging your workflows via our API.
-      </p>
-      <a
-        href="/get-started/get-api-key/"
-        class="btn"
-        >Generate API key
-        <span class="md-icon">{% include ".icons/material/arrow-right.svg" %}</span>
+
+  
+
+  <div class="cta-section">
+    <h2 class="cta-title">Built for developers by developers</h2>
+
+    <div class="button-grid">
+      <a href="/api-reference/reference/" class="button-card">
+        <span class="icon">{% include ".icons/octicons/code-16.svg" %}</span>
+        <div class="button-info">
+          <h3>API Reference</h3>
+          <p>Grab your API key to build and manage on kluster.ai</p>
```

Comment:
```suggestion
          <p>Grab your API key to build and manage on kluster.ai.</p>
```

I think we could switch this up to something about learning about the different endpoints and such that kluster.ai offers